test/input_gen/gen_layer_tests.py

```diff
@@ -909,6 +909,6 @@ def swiglu(inputs):
     record_single_fp16(
         swiglu_layer,
         [(2, 3, 3, 3), (2, 3, 3, 3)],
-        "swiglu",
+        "swiglufp16",
```

Comment:
I wonder why the input is float data type?

Reply:
As far as I recall, we have been formulating fp16 layer test data like this. This is because I found that using float16 as input makes tf to compute literally everything in fp16, resulting in too much precision loss or even worse, NaN values.

Reply:
Thanks.